tests/unit/plugins/service/test_init.py

```diff
@@ -71,9 +71,10 @@ def exp_sub(consumer_config):
      'projects/test-example/subscriptions/a-subscription'),
     (False, False, 'projects/test-example/topics/a-topic', 'a-subscription'),
 ])
-def test_get_event_consumer(local, provide_loop, topic, sub, consumer_config,
-                            exp_topic, auth_client, exp_sub, subscriber_client,
-                            emulator, monkeypatch, event_loop):
+def test_get_event_consumer_WARNING_EXPECTED(
```

Comment:
I would rather filter out an expected warning than have it in the name (see [pytest docs](https://docs.pytest.org/en/latest/warnings.html#pytest-mark-filterwarnings)).

Reply:
Seems like the spot generating the warning moved?  Fixed now.